commands/retry.js

```diff
@@ -13,15 +14,29 @@ function * run (context, heroku) {
   }
   let release = yield getLatestRelease()
 
+  let retry
```

Comment:
This makes me itch. Is there any way to write this as

```js
let retry = yield …
```

so that it's clearer that the value is determined by the sequence of side effects in the generator, rather than making that implicit?